packages/async-operations/src/async_operations.ts

```diff
@@ -130,6 +132,7 @@ export const startAsyncOperation = async (params: {
   stackName: string,
   systemBucket: string,
   useLambdaEnvironmentVariables?: boolean,
+  startEcsTaskFunc?: () => StartEcsTaskReturnType
```

Comment:
This is the magic which allows us to now pass in a mock for this function that starts ECS tasks. So for unit tests we can mock just this behavior and test that everything else for starting the async operation is setup properly